packages/react-components/src/components/VideoTile.tsx

```diff
@@ -221,6 +229,16 @@ const VideoTileMoreOptionsButton = (props: {
   );
 };
 
+/* @conditional-compile-remove(reaction) */
+const styleSheet = document.styleSheets[0];
+/* @conditional-compile-remove(reaction) */
+const keyframes1 = `@keyframes play {
+  from { background-position-y: 8568px; }
+  to { background-position-y: 0px; }
+}`;
+/* @conditional-compile-remove(reaction) */
+styleSheet.insertRule(keyframes1, styleSheet.cssRules.length);
```

Comment:
This dangerous as it is happening globally at import time and reliant on `document.styleSheets[0]` existing at this point in time.

It looks like fluent has a function for adding keyframes that does such checks: https://github.com/microsoft/fluentui/blob/master/packages/merge-styles/src/keyframes.ts with example usage: https://github.com/microsoft/fluentui/blob/master/packages/react/src/components/Spinner/Spinner.styles.tsx#L19

Can you try using that and defining in the videoTile.styles file?

Reply:
Will check this and update accordingly.